variables.tf

```diff
@@ -2,8 +2,42 @@
 # Input Variables
 ########################################################################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "watsonx_data_name" {
+  type        = string
+  description = "The name of the watsonx.data instance."
```

Comment:
This has no default value, meaning its a required input. However it would not be required if passing existing CRN right?

Reply:
Added a default value.